### PR TITLE
Remove svc_proxy module & tidy up views_enduser/urls_enduser modules

### PR DESCRIPTION
**Pull request by piglei**
- Remove svc_proxy module & tidy up views_enduser/urls_enduser modules
- While no API has been removed in the PR, these APIs starts with '/svc_workload/' will be removed in the near future
- "cnative.specs" module is not included, it has been processed in #1139

### CODE REVIEW

File: apiserver/paasng/paas_wl/bk_app/processes/urls.py [hunk 1]
```diff
@@ -16,24 +16,55 @@
...
+from . import views
+
+# In legacy architecture, all workloads's APIs starts with a fixed prefix "/svc_workloads" and use
+# a slightly different format. While the prefix is not required in the new architecture, we have
+# to keep it to maintain backward-compatibility.
+#
+# This function helps up to build paths with the legacy prefix.
+#
+# TODO: Remove the 'svc_workloads' prefix and clean up the URL paths.
+make_app_pattern_legacy_wl = partial(make_app_pattern, prefix="svc_workloads/api/processes/applications/")
```
Comment (shabbywu) [inline apiserver/paasng/paas_wl/bk_app/processes/urls.py:32]: 可以搞嵌套的.


```
from django.conf.urls import include, url
from django.urls import path



urlpatterns = [
    path("svc_workloads/", ([
        re_path(
            make_app_pattern(r"/processes/$"),
            views.ProcessesViewSet.as_view({"post": "update"}),
            name="api.processes",
        )
    ], None, None)
     # 分别是 urlpatterns, app_name, namespace
    )
]
```

File: apiserver/paasng/paas_wl/workloads/images/urls.py [hunk 2]
```diff
@@ -18,17 +18,29 @@
...
+
+
+# In legacy architecture, all workloads's APIs starts with a fixed prefix "/svc_workloads" and use
+# a slightly different format. While the prefix is not required in the new architecture, we have
+# to keep it to maintain backward-compatibility.
+#
+# This function helps up to build paths with the legacy prefix.
+#
+# TODO: Remove the 'svc_workloads' prefix and clean up the URL paths.
+def make_app_pattern_legacy_wl(suffix: str) -> str:
```
Comment (shabbywu) [inline apiserver/paasng/paas_wl/workloads/images/urls.py:31]: 同上。搞嵌套比现在这样写好看

File: apiserver/paasng/paas_wl/bk_app/processes/urls.py (hunk 1 above)
Comment (shabbywu) [reply apiserver/paasng/paas_wl/bk_app/processes/urls.py:32]: 觉得定义 urlpatterns 难看的话，可以保留个 legacy_urls.py 然后 include 进来。

File: apiserver/paasng/paasng/plat_admin/admin42/templates/admin42/applications/detail/engine/custom_domain.html [hunk 3]
```diff
@@ -73,9 +73,9 @@
     const envChoices = {{ env_choices | to_json }}
 
     const URLRouter = {
-        list: decodeURI("{% url 'workloads.proxy' 'admin42/applications/${code}/domains/' %}"),
```
Comment (shabbywu) [inline apiserver/paasng/paasng/plat_admin/admin42/templates/admin42/applications/detail/engine/custom_domain.html:76]: 有条件就全部改成 `url ${name}` 的形式吧。。。比硬编码写 url 靠谱多了。

such as `'{% url "admin.applications.detail.overview" "--placeholder--" %}'`

这里的 placeholder 是 `${code}` 和 `${id}`

Comment (piglei) [reply apiserver/paasng/paasng/plat_admin/admin42/templates/admin42/applications/detail/engine/custom_domain.html:76]: 工作量较大，本次不调整。

**Review by narasux — APPROVED** [review]

**Pull request merged**